src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/ReviewUnusedParametersTests.cs

```diff
@@ -926,14 +924,14 @@ public class C
     {{
     }}
 }}"
-                    }
+                    },
+                    AnalyzerConfigFiles = { ("/.editorconfig", $"[*]{Environment.NewLine}{editorConfigText}") }
```

Comment:
💡 A new line should be added at the end of this, specifically to avoid the case like this file encountered. Line 927 should not have changed as part of this pull request, but a change was forced because the line failed to include the trailing comma.